test/serve.ts

```diff
@@ -0,0 +1,55 @@
+import { expect } from 'chai';
+
+import { buildContext, mwInvoke } from './helpers';
+import { serveFiles, serve } from '../src/serve';
+
+describe('serveFiles', () => {
+  it('should return the requested file', async () => {
+    const ctx = buildContext(
+      '/assets/test.txt'
+    );
+
+    await mwInvoke(
+      serveFiles({
+        staticDir: `${process.cwd()}/test/assets`,
+      }),
+      ctx
+    );
+
+    expect(ctx.response.body instanceof Buffer).to.be.true;
```

Comment:
I suspect this should be:

```suggestion
    expect(ctx.response.body instanceof Buffer).to.be.true();
```

Reply:
Just `.true` is correct
https://www.chaijs.com/api/bdd/#method_true